### DISCUSSION TITLE
vuepress要怎么像vue一样，使用rem

### DISCUSSION THREAD

**Discussion by SouriChen** [vuepress/core#332] [Q&A] [answered]
在使用vuepress做静态化站点的时候，希望用到rem，但是没找到相关的配置方案

**Comment by Mister-Hope** [comment, marked as answer]
VuePress 没阻止你调用插件.....

你可以自行修改 bundlerConfig 去实现你想要的功能。你可以在 issue 和 dicussion 找到使用其他CSS插件的相关内容，可做参考。

总之我的意见是 VuePress 没有限制你这么做，怎么配置就要你自己去学了.....

**Reply by SouriChen** [reply]
是类似使用postcss-pxtorem等插件的方法
